app/scripts/update-translation.js

```diff
@@ -0,0 +1,98 @@
+import fs from 'node:fs/promises'
+import path from 'node:path'
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
+import { generateText } from 'ai';
+import { openai } from '@ai-sdk/openai';
+
+// Convert the module's URL to a file path
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+if (!process.env.OPENAI_API_KEY) {
+  console.error('Please set the OPENAI_API_KEY environment variable')
+  process.exit(1)
+}
+
+const model = openai('gpt-4');
+
+function stripQuotes(s) {
+  return s.replace(/^"(.*)"$/, '$1')
+}
+
+async function translateString(sourceLanguage, targetLanguage, key, sourceValue) {
+  const { text } = await generateText({
+    model: model,
+    system: `You are a climate tech translator concentrating on ${sourceLanguage}-${targetLanguage} translations. You return only the translated strings, no explanations or excuses. If you cannot translate the text, you return an empty string.`,
+    prompt: `Translate the following text from ${sourceLanguage} to ${targetLanguage} for the key ${key}:\n\n"${sourceValue}"`,
+  });
```

Comment:
For this simple task, your prompt will probably work fine.
Generally I try to structure the prompt in different sections.

1. Role of the model
2. Task
3. Description of input
4. Description of output and output format
5. Additional hints

So the system prompt could be rewritten to something along this line:
(But its usually more important for complex tasks)

> You are a climate tech translator specializing in ${sourceLanguage}-${targetLanguage} translations.
> 
> Your task is to translate text provided in the input. You handle only the translation of the text.
> 
> **Input:**
> You are provided with the input as a single string.
> 
> **Output:**
> The output must be a translation of the input string in the target language ${targetLanguage}.
> The output format is the pure translated string with no extra formatting or text. Do not provide explanations or excuses in your output.
>  If you cannot translate the text, return an empty string.

Reply:
I like it!

Reply:
Awesome! I'm adding this as an update task.